### PR TITLE
release 0.16.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
...
+- Upstream members are never collected [\#1274](https://github.com/voxpupuli/puppet-nginx/issues/1274)
+-  "location" directive is not allowed here in /etc/nginx/sites-enabled/example.mydomain.com.conf:2 [\#1271](https://github.com/voxpupuli/puppet-nginx/issues/1271)
+- Same location linked to multiple nginx::servers [\#1187](https://github.com/voxpupuli/puppet-nginx/issues/1187)
+- Nginx::Resource::Server: has no parameter named 'proxy\_send\_timeout' [\#1186](https://github.com/voxpupuli/puppet-nginx/issues/1186)
+
+**Merged pull requests:**
+
+- change rights for sites-enabled, streams-available [\#1289](https://github.com/voxpupuli/puppet-nginx/pull/1289) ([danwerspb](https://github.com/danwerspb))
+- update documentation and examples associated with adding upstream para… [\#1273](https://github.com/voxpupuli/puppet-nginx/pull/1273) ([alexskr](https://github.com/alexskr))
+- pin parallel\_tests for ruby 2.1.0 [\#1270](https://github.com/voxpupuli/puppet-nginx/pull/1270) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (ekohl) [inline CHANGELOG.md]: I'd skip this from the changelog since it doesn't really matter for users.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
...
+- Error pages on location level \[help\] [\#1279](https://github.com/voxpupuli/puppet-nginx/issues/1279)
+- Upstream members are never collected [\#1274](https://github.com/voxpupuli/puppet-nginx/issues/1274)
+-  "location" directive is not allowed here in /etc/nginx/sites-enabled/example.mydomain.com.conf:2 [\#1271](https://github.com/voxpupuli/puppet-nginx/issues/1271)
+- Same location linked to multiple nginx::servers [\#1187](https://github.com/voxpupuli/puppet-nginx/issues/1187)
+- Nginx::Resource::Server: has no parameter named 'proxy\_send\_timeout' [\#1186](https://github.com/voxpupuli/puppet-nginx/issues/1186)
+
+**Merged pull requests:**
+
+- change rights for sites-enabled, streams-available [\#1289](https://github.com/voxpupuli/puppet-nginx/pull/1289) ([danwerspb](https://github.com/danwerspb))
+- update documentation and examples associated with adding upstream para… [\#1273](https://github.com/voxpupuli/puppet-nginx/pull/1273) ([alexskr](https://github.com/alexskr))
```
Comment (ekohl) [inline CHANGELOG.md]: I've updated the title a bit

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
...
+
+- update location of passenger repo gpgkey [\#1277](https://github.com/voxpupuli/puppet-nginx/pull/1277) ([pauljflo](https://github.com/pauljflo))
+- Fix the condition for upstream members [\#1276](https://github.com/voxpupuli/puppet-nginx/pull/1276) ([SaschaDoering](https://github.com/SaschaDoering))
+
+**Closed issues:**
+
+- Error pages on location level \[help\] [\#1279](https://github.com/voxpupuli/puppet-nginx/issues/1279)
+- Upstream members are never collected [\#1274](https://github.com/voxpupuli/puppet-nginx/issues/1274)
+-  "location" directive is not allowed here in /etc/nginx/sites-enabled/example.mydomain.com.conf:2 [\#1271](https://github.com/voxpupuli/puppet-nginx/issues/1271)
+- Same location linked to multiple nginx::servers [\#1187](https://github.com/voxpupuli/puppet-nginx/issues/1187)
```
Comment (ekohl) [inline CHANGELOG.md]: I've added skip-changelog since a PR implements this. Then it's redundant.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
...
+
+**Fixed bugs:**
+
+- update location of passenger repo gpgkey [\#1277](https://github.com/voxpupuli/puppet-nginx/pull/1277) ([pauljflo](https://github.com/pauljflo))
+- Fix the condition for upstream members [\#1276](https://github.com/voxpupuli/puppet-nginx/pull/1276) ([SaschaDoering](https://github.com/SaschaDoering))
+
+**Closed issues:**
+
+- Error pages on location level \[help\] [\#1279](https://github.com/voxpupuli/puppet-nginx/issues/1279)
+- Upstream members are never collected [\#1274](https://github.com/voxpupuli/puppet-nginx/issues/1274)
```
Comment (ekohl) [inline CHANGELOG.md]: I've added skip-changelog since a PR implements this. Then it's redundant.

**Comment by Infraded** [comment]
@bastelfreak It doesn't look like this release ever actually got cut. Any chance this could be pushed out, or is there another release coming up?

**Comment by bastelfreak** [comment]
Hi, I will prepare a new release later

**Pull request merged**